PlaidLink.tsx

```diff
@@ -64,16 +64,16 @@ export const openLink = async (props: PlaidLinkProps) => {
             props.onExit(data);
           }
         } else {
-          switch (result.metadata.status) {
+          switch (result.status) {
```

Comment:
As chatted earlier it is probably be best to not rely on status, but use the type of result to determine whether `onSuccess` or `onExit` should be called.

Reply:
Agreed, and now we have types!